cloud-aws-cloudformation/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsConnector.java

```diff
@@ -63,7 +62,7 @@
     private AwsPlatformResources awsPlatformResources;
 
     @Inject
-    private AwsSetup awsSetup;
+    private AwsLegacySetup awsSetup;
```

Comment:
AwsCloudformationSetup seems to be a better name :D

Reply:
I will rename it under this ticket: https://jira.cloudera.com/browse/CB-13032